packages/create-atomic/src/plopfile.ts

```diff
@@ -72,15 +72,42 @@ export default function (plop: NodePlopAPI) {
           '../templates/tsconfig.json',
           '../templates/package.json',
           '../templates/webpack.config.mjs',
+          '../templates/netlify.toml',
         ],
       },
+      function installPackagesPrompt() {
+        return 'Installing packages 🚀';
+      },
       function installPackages(answers) {
-        const {project} = answers as PromptsAnswers;
-        spawn(getPackageManager(), ['install'], {
-          stdio: 'inherit',
-          cwd: `${currentPath}/${project}/`,
+        return new Promise((resolve, reject) => {
```

Comment:
swtiched to a promise in order to wait until displaying the next message

Reply:
Ideally, it'd be nice to have a little spinny boi or progression bar (probably the former because we don't have a good measure of progression) to indicate we didn't crash.